numpy/_core/fromnumeric.py

```diff
@@ -3514,13 +3514,13 @@ def mean(a, axis=None, dtype=None, out=None, keepdims=np._NoValue, *,
 
 
 def _std_dispatcher(a, axis=None, dtype=None, out=None, ddof=None,
-                    keepdims=None, *, where=None, mean=None):
+                    keepdims=None, *, where=None, mean=None, correction=None):
     return (a, where, out, mean)
 
 
 @array_function_dispatch(_std_dispatcher)
 def std(a, axis=None, dtype=None, out=None, ddof=0, keepdims=np._NoValue, *,
-        where=np._NoValue, mean=np._NoValue):
+        where=np._NoValue, mean=np._NoValue, correction=0):
```

Comment:
Should we make the default `correction=_np.NoValue` so that we can truly recognize if some value is put in? Could even consider doing the same for `ddof`, but that would be an API change for people whose classes have a `.std()` method, so probably best not.

Reply:
I like this idea - done! Then `std(x, ddof=1, correction=0)` can also be recognized as incorrect set of parameters.